odbc-sys/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "odbc-sys"
+version = "0.21.4"
```

Comment:
This will have to be set to `0.0.0` for the version check to pass. The decision was made to ensure all crates stayed at version `0.0.0`. That or we need to get rid of that test.